tests/models/llava_next_video/test_modeling_llava_next_video.py

```diff
@@ -430,13 +430,13 @@ def test_small_model_integration_test(self):
 
         # verify generation
         output = model.generate(**inputs, do_sample=False, max_new_tokens=40)
-        EXPECTED_DECODED_TEXT = 'USER: \nWhy is this video funny? ASSISTANT: The humor in this video comes from the unexpected and somewhat comical situation of a young child reading a book while another child is attempting to read the same book. The child who is reading the book seems'  # fmt: skip
-
-        self.assertEqual(
-            self.processor.decode(output[0], skip_special_tokens=True),
-            EXPECTED_DECODED_TEXT,
+        EXPECTED_DECODED_TEXT = (
+            "USER: \nWhy is this video funny? ASSISTANT: The humor in this video comes from the unexpected and somewhat comical situation of a young child reading a book while another child is attempting to read the same book. The child who is reading the book seems",  # cuda output
+            "USER: \nWhy is this video funny? ASSISTANT: The humor in this video comes from the unexpected and somewhat comical situation of a young child reading a book while wearing a pair of glasses that are too large for them. The glasses are",  # xpu output
         )
 
+        self.assertTrue(self.processor.decode(output[0], skip_special_tokens=True) in EXPECTED_DECODED_TEXT)
```

Comment:
is this expected obtained from running on xpu device? We should not change in this way as different hardware may need different values.

Reply:
Yes, the output may not exactly the same even if both cuda and XPU use float16 precision. Please let me know if you have better options. Thanks!

Reply:
We are using Nvidia T4 as the major CI environment. We are currently doing like this

https://github.com/huggingface/transformers/blob/69bc848480d5f19a537a70ce14f09816b00cd80f/tests/models/gemma/test_modeling_gemma.py#L688-L701

which is quite limited and not applicable here. @ivarflakstad is working on #36569 to improve this part of logic.

In the meantime, I will check on T4 and see what has to be updated in order to merge

Reply:
OK, please let me know if there is anything I can do.

Reply:
it works too! Thank you